samples/bot-conversation/nodejs/appManifest/manifest.json

```diff
@@ -58,6 +58,26 @@
             {
               "title": "MessageAllMembersUsingAadId",
               "description": "Send 1 to 1 message to all members of the current conversation using their AADId"
+            },
+            {
+              "title": "AI label",
+              "description": "AI label added to your bot message specifies that the message is generated by AI"
+            },
+            {
+              "title": "sensitivity",
+              "description": "Sensitivity label added to your bot message specifies the confidentiality of a message"
+            },
+            {
+              "title": "Feedback buttons",
+              "description": "Feedback buttons that helps to provide feedback for a bot message"
+            },
+            {
+              "title": "citation",
+              "description": "Used to cite the sources of the bot message"
+            },
+            {
+              "title": "SendAITextMessage",
+              "description": "Sends an format AI text message using bot"
```

Comment:
Sends an AI generated message in text format using bot

Reply:
Fixed